CMakeLists.txt

```diff
@@ -520,13 +520,9 @@ elseif(BUILD_HYDRA_CORE)
     target_link_libraries(Alber PUBLIC AlberCore)
 elseif(BUILD_LIBRETRO_CORE)
     include_directories(third_party/libretro/include)
-    add_library(Alber SHARED src/libretro_core.cpp)
```

Comment:
The target name should probably be kept the same as it was before, since the entire CMake is built on the idea that Alber = Frontend + Core, so other parts will refer to it as such.